src/platform-includes/getting-started-config/javascript.react.mdx

```diff
@@ -10,13 +11,21 @@ Sentry.init({
     new Sentry.BrowserTracing({
       // Set `tracePropagationTargets` to control for which URLs distributed tracing should be enabled
       tracePropagationTargets: ["localhost", /^https:\/\/yourserver\.io\/api/],
+      // See docs for support of different versions of variation of react router
+      // https://docs.sentry.io/platforms/javascript/guides/react/configuration/integrations/react-router/
+      routingInstrumentation: Sentry.reactRouterV6Instrumentation(
+        React.useEffect,
+        useLocation,
+        useNavigationType,
+        createRoutesFromChildren,
+        matchRoutes
+      ),
     }),
     new Sentry.Replay()
   ],
 
   // Set tracesSampleRate to 1.0 to capture 100%
   // of transactions for performance monitoring.
-  // We recommend adjusting this value in production
```

Comment:
@smeubank - what's the recommended phrasing on this?

Most SDKs have a comment to this effect for `tracesSampleRate`

Reply:
i don't think there is a standard phrasing, and there's some mixed messages internally setting to 100% for many/most users will result in problems in quota at least